src/main/java/com/atlassian/jira/cloud/jenkins/config/JiraCloudPluginConfig.java

```diff
@@ -77,6 +82,23 @@ public boolean configure(final StaplerRequest req, final JSONObject json) throws
                 }
             }
 
+            if (json.containsKey(FIELD_NAME_SITES2)) {
```

Comment:
Pretty much duplicated everything in this file that has to do with the `sites` field and made it work with the `sites2` field.